app/utils/Navigator.scala

```diff
@@ -194,7 +194,7 @@ class Navigator @Inject extends Logging with FeatureSwitching {
     InternationalActivitiesId -> { userAnswers =>
       userAnswers.businessEntity match {
         case Some(_) if userAnswers.internationalActivities.contains(true) => pageIdToPageLoad(EligibilityDropoutId(InternationalActivitiesId.toString))
-        case Some(UKCompany) => pageIdToPageLoad(InvolvedInOtherBusinessId)
+        case Some(UKCompany | LimitedLiabilityPartnership) => pageIdToPageLoad(InvolvedInOtherBusinessId)
```

Comment:
This will need to be separated from the UK company case and have a PartnershipFlow feature switch guard on it so the routing defaults to the exception page if the FS is disabled. The UK Company flow is always there because it's been there the longest